src/pages/webdev.astro

```diff
@@ -26,78 +26,3 @@ const { frontmatter } = Astro.props;
 				body="This Blog series is writted by Mr. Ånand while learning Javascript."
 			/>
 			<Card
```

Comment:
- The Web Dev resource you added has been deleted under the last merge commit from main branch. 
- Please add your resource again to the `webdev.astro` file and submit it.